src/main/java/com/kkm/kkm_server_v2/domain/post/domain/Post.java

```diff
@@ -65,6 +67,12 @@ public void updateStatus(PostStatus status) {
 
     private boolean ripped; // 찢어짐
 
+    private LocalDateTime pullDate;
+    public void updatePullDate() {
```

Comment:
```suggestion
    public void pull() {
```

'끌어올리기 날짜 수정하기'라는 함수 이름 보다는 '끌어올리기'로 하는 건 어떨까용?

Reply:
굿뜨